docs/sources/mimir/migration-guide/migrating-from-cortex.md

```diff
@@ -211,13 +215,13 @@ You can update to the Grafana Mimir Helm chart from the Cortex Helm chart.
 
    b. Install the recording and alerting rules into the ruler or a Prometheus server.
 
-1. Run the following command to add the Grafana Helm chart repository:
+2. Run the following command to add the Grafana Helm chart repository:
```

Comment:
Please use `1. ` for each number if feasible, so that numbered steps are more future proofed. (It might not be feasible.)

Reply:
Oh I just got to know if markdown will automatically handle the numbering step. Will update this.